lib/versions.js

```diff
@@ -48,14 +58,6 @@ module.exports = function(github, opts) {
         return tag;
     }
 
-    // Extract channel of version
-    function extractChannel(tag) {
-        var suffix = tag.split('-')[1];
-        if (!suffix) return 'stable';
-
-        return suffix.split('.')[0];
-    }
-
```

Comment:
I think you may have moved this out of the closure to allow for testing. Is there another way to to it? Would be nice not to have to move it. Maybe we could test the function that calls into `extractChannel` instead? idk.